### ISSUE TITLE
How to check recipes

### ISSUE THREAD

**Issue by asdf-hash** [ArsenicPurple/CreateArmsRace#8]
i might be dumb but how do i check the recipes emi doesnt work when clicking on the guns

**Comment by ArsenicPurple** [comment]
I don't believe EMI has support for mechanical crafters currently. I know JEI works with Create recipes. REI also works but you need a compatibility addon.
